### DISCUSSION TITLE
How do I run google calender api on a symphony custom bundle?

### DISCUSSION THREAD

**Discussion by seleondevel** [symfony/symfony#57221] [Q&A]
Hello,

How do I run google calender api on a symphony custom bundle?

**Comment by Nek-** [comment]
Hello, it's Symfony not symphony. 😄 

Without more information about your question, you will get no response. What bundle? In what way is it custom? Is your question even related to Symfony or is it more php-general on what kind of dependency you should use (in this case StackOverflow is a better place)

**Comment by seleondevel** [comment]
https://github.com/googleapis/google-api-php-client

https://github.com/googleworkspace/php-samples/tree/main/calendar/quickstart

#[AsEntityListener(entity: Timesheet::class, event: Events::postPersist, method: 'postPersist')]

Using the above Google calender api code, I'm trying to call Google calender api from the Doctorine entity lister, but I have to set redirect_uri and do routing, but I don't know how to do it.

**Comment by seleondevel** [comment]
Administrator, I've already fixed this issue! Please delete this discussion post!
